ghost/core/core/frontend/src/ghost-stats/ghost-stats.js

```
@@ -0,0 +1,297 @@
+      'api_token',
+      'email',
+      'address',
+      'phone',
+      'sex',
+      'gender',
+      'order',
+      'order_id',
+      'orderid',
+      'payment',
+      'credit_card',
+    ]
```

ESLint

[error] Expected indentation of 8 spaces but found 4.

(indent)